com.ampl.ide.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Spaziale -->
+<component type="desktop-application">
+  <id>com.ampl.ide</id>
+  <launchable type="desktop-id">com.ampl.ide.desktop</launchable>
+  <name>AMPL IDE</name>
+  <developer_name>AMPL Optimization, Inc</developer_name>
+  <summary>A simple and straightforward enhanced modeling interface for AMPL users.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Is redistribution allowed?

Reply:
No, but I used extra data. Also I think they are unlikely to respond.